package.json

```diff
@@ -91,13 +92,16 @@
     "mocha": "^1.21.4",
     "open": "0.0.5",
     "phantomjs": "^1.9.18",
+    "querystringify": "0.0.3",
     "raw-loader": "^0.5.1",
     "redis": "^0.12.1",
     "requirejs": "^2.1.20",
     "run-sequence": "^1.1.4",
     "sinon": "^1.12.1",
     "string-replace-webpack-plugin": "0.0.3",
     "uglify-js": "^2.4.15",
+    "url-parse": "^1.0.5",
+    "webdriverio": "^3.2.5",
```

Comment:
@bjorn-wo 

Can you remove this dev dependency as we aren't setup yet for automated testing.